app.rb

```diff
@@ -0,0 +1,23 @@
+require 'sinatra/base'
+require 'pg'
+require_relative './lib/peep'
+
+class App < Sinatra::Base
+
+  attr_reader :content, :author
```

Comment:
What is this line doing? Think this might be accidental repetition from the peep class.